build/Wikibase/LocalSettings.wbs-extensions.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
should we maybe just call it `LocalSettings.wbs.php`? It is doing more than just the extensions.

Reply:
agreed, and updated.